app/src/main/java/com/starry/myne/ui/theme/Type.kt

```diff
@@ -21,49 +21,105 @@ import androidx.compose.ui.text.PlatformTextStyle
 import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.text.font.Font
 import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontStyle
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.LineHeightStyle
 import androidx.compose.ui.unit.sp
 import com.starry.myne.R
 
 
-val figeronaFont = FontFamily(
+val poppinsFont = FontFamily(
     fonts = listOf(
         Font(
-            resId = R.font.figerona_black,
-            weight = FontWeight.Black
+            resId = R.font.poppins_black,
+            weight = FontWeight.Black,
+            style = FontStyle.Normal
         ),
         Font(
-            resId = R.font.figerona_bold,
-            weight = FontWeight.Bold
+            resId = R.font.poppins_bold,
+            weight = FontWeight.Bold,
+            style = FontStyle.Normal
         ),
         Font(
-            resId = R.font.figerona_extrabold,
-            weight = FontWeight.ExtraBold
+            resId = R.font.poppins_extrabold,
+            weight = FontWeight.ExtraBold,
+            style = FontStyle.Normal
         ),
         Font(
-            resId = R.font.figerona_extralight,
-            weight = FontWeight.ExtraLight
+            resId = R.font.poppins_extralight,
+            weight = FontWeight.ExtraLight,
+            style = FontStyle.Normal
         ),
         Font(
-            resId = R.font.figerona_light,
-            weight = FontWeight.Light
+            resId = R.font.poppins_light,
+            weight = FontWeight.Light,
+            style = FontStyle.Normal
         ),
         Font(
-            resId = R.font.figerona_medium,
-            weight = FontWeight.Medium
+            resId = R.font.poppins_medium,
+            weight = FontWeight.Medium,
+            style = FontStyle.Normal
         ),
         Font(
-            resId = R.font.figerona_regular,
-            weight = FontWeight.Normal
+            resId = R.font.poppins_regular,
+            weight = FontWeight.Normal,
+            style = FontStyle.Normal
         ),
         Font(
-            resId = R.font.figerona_semibold,
-            weight = FontWeight.SemiBold
+            resId = R.font.poppins_semibold,
+            weight = FontWeight.SemiBold,
+            style = FontStyle.Normal
         ),
         Font(
-            resId = R.font.figerona_thin,
-            weight = FontWeight.Thin
+            resId = R.font.poppins_thin,
+            weight = FontWeight.Thin,
+            style = FontStyle.Normal
+        ),
+
+        Font(
```

Comment:
Please remove these additional italic fonts, as they aren't used anywhere and will just increase the size of the APK file.

Reply:
What about text in italics inside a book? Is that supported?

Reply:
> What about text in italics inside a book? Is that supported?

No, it currently doesn't take the text formatting of each letter into account. It focuses on extracting text content from HTML/XHTML files inside the EPUB archive and displays it in whatever font the user has selected.

Reply:
Indeed italics don't seem to be supported. Maybe we could keep the font anyway, to have it future proof?

![Screenshot_20240923-142617.png](https://github.com/user-attachments/assets/00638cad-8a2c-4143-9e02-546da64f25c0)

![Screenshot_20240923-142609.png](https://github.com/user-attachments/assets/aef32afe-8b04-47ad-ae70-ffd724a819bd)

Reply:
> Maybe we could keep the font anyway, to have it future proof?

We can always add them back if or when needed, either from Google Fonts or by going back to this commit and picking up the files. However, I don't think it would be a good idea to increase the APK size by a few megabytes over this for the time being, as it would waste users' bandwidth and potentially their storage.

Personally, I prefer to keep the APK size as small as possible, I hope you understand :)